src/app/modules/users/components/users-list/users-list.component.html

```diff
@@ -0,0 +1,26 @@
+<table *ngIf="users" class="table table-sm table-bordered table-striped mb-0" datatable [dtTrigger]="dtTrigger">
+  <thead class="thead-blue">
+    <tr class="d-flex">
+      <th class="col-4">User Email</th>
+      <th class="col-4">Names</th>
+      <th class="col-4 text-center">Admin Role</th>
+    </tr>
+  </thead>
+  <app-loading-bar *ngIf="isLoading$ | async"></app-loading-bar>
+  <tbody *ngIf="(isLoading$ | async) === false">
+    <tr class="d-flex" *ngFor="let user of users">
+      <td class="col-sm-4">{{ user.email }}</td>
+      <td class="col-sm-4">{{ user.name }}</td>
+      <td class="col-sm-4 text-center custom-control custom-switch">
```

Comment:
This is going to be confusing. Let's either make this column invisible or remove it completely.

Reply:
Ok, I will remove them, that option will enter in another ticket.